Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -892,6 +903,44 @@ - (void)_createAndConfigureLongPressGestureRecognizer
     [self addGestureRecognizer:_longPressGestureRecognizer.get()];
 }
 
+- (BOOL)_shouldUseUIContextMenuAsyncConfiguration
+{
+#if HAVE(UI_CONTEXT_MENU_ASYNC_CONFIGURATION)
+    if (!_page->preferences().useAsyncUIKitInteractions())
+        return NO;
+
+    static BOOL hasAsyncConfigurationClass = ^{
+        return !!get_UIContextMenuAsyncConfigurationClass();
+    }();
```

Comment:
Good catch — for some reason, I thought it would complain about it not being a compile-time expression.

Changed to:

```
    static BOOL hasAsyncConfigurationClass = !!get_UIContextMenuAsyncConfigurationClass();
    return hasAsyncConfigurationClass;
```